app/src/main/AndroidManifest.xml

```diff
@@ -10,14 +10,22 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+  <!--      <activity android:name=".MainActivity"
```

Comment:
IMO you can remove commented code if you don't think we will need it in the future.